### PR TITLE
Industry 4.0 - Motivation for OPC UA Binding

### PR DESCRIPTION
**Pull request by sebastiankb**

### CODE REVIEW

**Comment by sebastiankb** [comment]
The text also include a link to a draft document about the joint activity.

**Comment by sebastiankb** [comment]
I think this PR does not make sense to be part of the use case document. As discussed in the liaison call, we should work on a common use case document together with the OPCF.

**Comment by mlagally** [comment]
UC call on 21.9.
This  contribution contains proposed next steps / joint activity between different SDOs, which go beyond the scope of a use case.
Please consider reworking it to make the Industry 4.0 scenario the central use case and define OPC-UA binding/profile as a requirement.
Some parts of this document could go into a requirements document / section.

**Comment by sebastiankb** [comment]
Ready for review

File: USE-CASES/use-case-opc-ua-binding.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+none
+
+### Internationalisation (i18n) Considerations:
+
+OPU-UA data model contains some places to provide human-readable text (e.g., browse name). This should be also reflected in the Thing Description with the correct language context. 
+
+### Requirements:
+
+An OPC UA binding for Web of Things needs an own set of OPC-UA specific vocabulary definitions which should be developed together with the experts from the OPC Foundation. Also see the [liason](https://opcfoundation.org/news/opc-foundation-news/w3c-and-opcf-to-integrate-opc-ua-into-the-web-of-things/).
```
Comment (mmccool) [inline USE-CASES/use-case-opc-ua-binding.md:79]: liaison

File: USE-CASES/use-case-opc-ua-binding.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Based on the bottling line's current power consumption, which is measured via Modbus, the backend system decides to increase productivity when surplus renewable energy is available.
+
+In doing so, the backend system interacts via OPC-UA to release the 4 fillers of the filling module and increases the speed of the transport system.
+
+If the backend system detects that less renewable energy is being produced, it will initiate standard production and reduce the transport speed and return the 2 fillers of the filling module.
+
+
+### Security Considerations:
+
+OPC-UA has different security modes (sign and/or encrypted, policies, and authentication). Those should be addressed and described in Thing Descriptions with a standardized vocabulary definition.
```
Comment (mmccool) [inline USE-CASES/use-case-opc-ua-binding.md:63]: additional considerations may apply if a web bridge is created using WoT servients

File: USE-CASES/use-case-opc-ua-binding.md (hunk 1 above)
Comment (sebastiankb) [reply USE-CASES/use-case-opc-ua-binding.md]: done

File: USE-CASES/use-case-opc-ua-binding.md (hunk 2 above)
Comment (sebastiankb) [reply USE-CASES/use-case-opc-ua-binding.md]: new version covers this points. thanks

**Review by mlagally — APPROVED** [review]

**Pull request merged**